phpcs.xml.dist

```diff
@@ -1,121 +1,257 @@
-				<element value="textContent" />
-				<element value="parentNode" />
-				<element value="nodeType" />
-				<element value="nodeName" />
-				<element value="nextSibling" />
-				<element value="previousSibling" />
-				<element value="firstChild" />
-				<element value="lastChild" />
-				<element value="nodeValue" />
-				<element value="ownerElement" />
-				<element value="documentElement" />
-				<element value="removeChild" />
-				<element value="ownerDocument" />
-				<element value="strictErrorChecking" />
-				<element value="preserveWhiteSpace" />
-			</property>
-		</properties>
-	</rule>
-
-	<rule ref="VariableAnalysis.CodeAnalysis.VariableAnalysis" />
-
-	<!-- Check for cross-version support for PHP 7.2 and higher. -->
-	<config name="testVersion" value="7.2-" />
-	<rule ref="PHPCompatibilityWP" />
-
-	<!-- The main plugin file should be parsable by PHP < 7.0. -->
-	<rule ref="Generic.Arrays.DisallowLongArraySyntax.Found">
-		<exclude-pattern>web-stories.php</exclude-pattern>
-		<exclude-pattern>includes/compat/Web_Stories_Compatibility.php</exclude-pattern>
-	</rule>
-
-	<rule ref="WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid">
-		<exclude-pattern
+    <exclude-pattern>tests/*</exclude-pattern>
+  </rule>
+
+  <rule ref="WordPress.WP.EnqueuedResources">
+    <type>error</type>
+    <exclude-pattern>tests/*</exclude-pattern>
+  </rule>
+
+  <rule ref="WordPress.WP.I18n">
+    <properties>
+      <property name="text_domain" type="array">
+        <element value="web-stories" />
+      </property>
+    </properties>
+  </rule>
+
+  <rule ref="WordPress.NamingConventions.ValidVariableName">
+    <properties>
+      <property name="customPropertiesWhitelist" type="array">
+        <element value="childNodes" />
+        <element value="tagName" />
+        <element value="textContent" />
+        <element value="parentNode" />
+        <element value="nodeType" />
+        <element value="nodeName" />
+        <element value="nextSibling" />
+        <element value="previousSibling" />
+        <element value="firstChild" />
+        <element value="lastChild" />
+        <element value="nodeValue" />
+        <element value="ownerElement" />
+        <element value="documentElement" />
+        <element value="removeChild" />
+        <element value="ownerDocument" />
+        <element value="strictErrorChecking" />
+        <element value="preserveWhiteSpace" />
+      </property>
+    </properties>
+  </rule>
+
+  <rule ref="VariableAnalysis.CodeAnalysis.VariableAnalysis" />
+
+  <!-- Check for cross-version support for PHP 7.2 and higher. -->
+  <config name="testVersion" value="7.2-" />
+  <rule ref="PHPCompatibilityWP" />
+
+  <!-- The main plugin file should be parsable by PHP < 7.0. -->
+  <rule ref="Generic.Arrays.DisallowLongArraySyntax.Found">
+    <exclude-pattern>web-stories.php</exclude-pattern>
+    <exclude-pattern
+    >includes/compat/Web_Stories_Compatibility.php</exclude-pattern>
```

Comment:
?

Reply:
I think this comes from Prettier. I ran everything through `npx prettier --write phpcs.xml.dist`